apps/client/src/index.js

```diff
@@ -29,6 +32,14 @@ const AppGlobalStyle = createGlobalStyle`
   }
 `;
 
+Sentry.init({
+  dsn: "https://2996729fb40e46d18b20f85cd57b4dd3@sentry.data.amsterdam.nl/49",
+  release: process.env.REACT_APP_VERSION,
+  environment: document.domain,
+  integrations: [new ApmIntegrations.Tracing()],
+  tracesSampleRate: 0.25, // must be present and non-zero
+});
+
```

Comment:
Can you change this to `Sentry.init(sentryConfig);` and move the config to a new folder `config/sentry.ts`? 

Let's make a new ticket to move MatomoConfig and the app config there as well